src/pages/STG_PanelContactRoles.page

```diff
@@ -1,17 +1,17 @@
-<apex:page controller="STG_PanelContactRoles_CTRL" >
-    <apex:form id="form" styleClass="slds-m-around--x-large">
+<apex:page controller="STG_PanelContactRoles_CTRL" docType="html-5.0" standardStylesheets="false" >
```

Comment:
Increased padding needed on this page as well.

Reply:
Fixed, though this required wrapping all instances of the `slds-section-title_divider` style div in another div with the `slds-section` class. It appears to be another change in the overall slds styling between the old version the page used to use and the new version.